### PR TITLE
Celestial serialization

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: ksp_plugin/celestial.hpp [hunk 1]
```diff
@@ -28,18 +29,23 @@ class Celestial {
...
+
   MassiveBody const& body() const;
   bool has_parent() const;
   Celestial const& parent() const;
+  void set_parent(not_null<Celestial const*> const parent);
+
+  // The following accessors require |is_initialized()|.
   Trajectory<Barycentric> const& history() const;
+  not_null<Trajectory<Barycentric>*> mutable_history();
   Trajectory<Barycentric> const& prolongation() const;
```
Comment (pleroy) [inline ksp_plugin/celestial.hpp]: Blank lines and comments just line in Vessel.

File: ksp_plugin_test/celestial_test.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+TEST_F(CelestialDeathTest, SerializationError) {
+  EXPECT_DEATH({
+    serialization::Celestial message;
+    celestial_->WriteToMessage(&message);
+  }, "is_initialized");
+}
+
+TEST_F(CelestialTest, SerializationSuccess) {
+  serialization::Celestial message;
+  EXPECT_FALSE(message.has_history_and_prolongation());
```
Comment (pleroy) [inline ksp_plugin_test/celestial_test.cpp]: I wouldn't do this, it's guaranteed by the protobuf library.

File: ksp_plugin_test/celestial_test.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  celestial_->CreateHistoryAndForkProlongation(t1_, d1_);
+  message.Clear();
```
Comment (pleroy) [inline ksp_plugin_test/celestial_test.cpp]: Same, remove.

File: ksp_plugin/celestial.hpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/celestial.hpp]: Done.

File: ksp_plugin_test/celestial_test.cpp (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_test/celestial_test.cpp]: Done.

File: ksp_plugin_test/celestial_test.cpp (hunk 3 above)
Comment (eggrobin) [inline ksp_plugin_test/celestial_test.cpp]: Done.

**Pull request merged**